### ISSUE TITLE
Re-initialize few corrupt canisters

### ISSUE THREAD

**Issue by ravibazz** [go-bazzinga/hot-or-not-backend-canister#201]
There are few canisters that are corrupted over time. Re-initializethe canisters that are corrupt with fresh new wasm.

**Comment by ravibazz** [comment]
Few of the known canisters that have been corrupted: 
```
failed_canister_ids = vec {
      record {
        principal "7by5m-4aixc-gsdmp-kp25f-gazin-kkxrk-3kqkt-bfgby-jo7fy-te5nd-tqe";
        principal "6l6jz-kaaaa-aaaao-actmq-cai";
        "Attempt to execute a message on canister 6l6jz-kaaaa-aaaao-actmq-cai which contains no Wasm module";
      };
      record {
        principal "trexp-i2v4t-wb7uj-jr7mz-bvugi-zna7s-lb42p-hufc7-edii3-6topz-sae";
        principal "rw62l-xyaaa-aaaao-aayca-cai";
        "Attempt to execute a message on canister rw62l-xyaaa-aaaao-aayca-cai which contains no Wasm module";
      };
      record {
        principal "fxfxj-ws64p-fiejm-qf6hu-wnp3x-vcdwf-fn3zi-moxr6-b3x5w-z2a5f-gae";
        principal "4thmb-taaaa-aaaao-aavfq-cai";
        "Attempt to execute a message on canister 4thmb-taaaa-aaaao-aavfq-cai which contains no Wasm module";
      };
      record {
        principal "ct2qv-qftza-d4vci-xk7x4-hunub-obedf-grfyl-2jj6f-f4o2u-qnurr-gae";
        principal "du74r-syaaa-aaaao-aa47q-cai";
        "Attempt to execute a message on canister du74r-syaaa-aaaao-aa47q-cai which contains no Wasm module";
      };
      record {
        principal "ep6m3-gxnuh-rrtge-i3upr-ugqvh-ucsjl-w3brb-m5tlb-lmtkh-iognc-5qe";
        principal "u6qff-cqaaa-aaaao-aczfa-cai";
        "Attempt to execute a message on canister u6qff-cqaaa-aaaao-aczfa-cai which contains no Wasm module";
      };
```

**Comment by harshbaz** [comment]
Can you recheck these IDs as well?

```
      '6l6jz-kaaaa-aaaao-actmq-cai'
      'rw62l-xyaaa-aaaao-aayca-cai'
      '4thmb-taaaa-aaaao-aavfq-cai'
      'du74r-syaaa-aaaao-aa47q-cai'
      'u6qff-cqaaa-aaaao-aczfa-cai'
      'p7ngy-hyaaa-aaaao-aesjq-cai'
      'ynvsv-3qaaa-aaaao-ae2gq-cai'
      '4xvw2-giaaa-aaaao-aejwq-cai'
      'nut4d-faaaa-aaaao-agaxa-cai'
      'raidg-jaaaa-aaaao-afbxa-cai'
```

**Comment by ravibazz** [comment]
Closed in #224
